common/eth2/src/types.rs

```diff
@@ -888,7 +887,6 @@ pub struct SseBlock {
 }
 
 #[derive(PartialEq, Debug, Serialize, Deserialize, Clone)]
-// TODO(pawan): modify spec to maybe include proofs too?
```

Comment:
Since we don't send the full block in the block event, I think it's ok to omit the inclusion proofs

Reply:
Ohh did not realise this, seems good then